src/js/techmix_sector.js

```diff
@@ -0,0 +1,378 @@
+import * as d3 from 'd3';
+import { union } from 'd3-array';
+import '../css/plot_styles.css';
+
+export class techmix_sector {
+	constructor(container, data) {
+		// Data needs to be ordered on year (increasing)
+		// Data needs to have two years in
```

Comment:
Comment seems to be missing crucial info?

Reply:
What do you mean? These are comments I put in about data requirements for this function. They are not exhaustive though. During documenting/refactoring stage we should probably put in tests to check if all the conditions for data are met. At the same time this function is not meant to be used by general users like functions in R packages so I am not sure to what level of detail we should document it.

Reply:
"Data needs to have two years in" seems like it isn't a complete sentence. Two years in ... what?

Reply:
forgive my English :D I'll make it clearer